advisories/github-reviewed/2023/07/GHSA-58qw-p7qm-5rvh/GHSA-58qw-p7qm-5rvh.json

```diff
@@ -94,7 +94,7 @@
               "introduced": "0"
             },
             {
-              "fixed": "9.4.52"
+              "fixed": "9.4.52.v20230823"
```

Comment:
This is the only line I intended to change through the form available at:
https://github.com/advisories/GHSA-58qw-p7qm-5rvh/improve